predict.py

```diff
@@ -11,6 +12,7 @@
 
 ABEJA_STORAGE_DIR_PATH = os.getenv('ABEJA_STORAGE_DIR_PATH', '~/.abeja/.cache')
 ABEJA_TRAINING_RESULT_DIR = os.getenv('ABEJA_TRAINING_RESULT_DIR', 'abejainc_training_result')
+Path(ABEJA_TRAINING_RESULT_DIR).mkdir(exist_ok=True)
```

Comment:
this is what i concerned before..., 
sorry for this...

Reply:
No it's not your fault.
This is our own mistake. Anyway thank you for reviewing!